drivers/ZXT-120/device.js

```diff
@@ -45,12 +45,47 @@ class ZXT120 extends ZwaveDevice {
 			    'Manufacturer Data': new Buffer([0]),
 			})
 		});
+
+		// this.registerCapability('FAN_mode', 'THERMOSTAT_FAN_MODE', {
+		// 	get: 'THERMOSTAT_FAN_MODE_GET',
+		// 	getOpts: {
+		// 		//getOnStart: true,
+		// 	},
+		// 	set: 'THERMOSTAT_FAN_MODE_SET',
+		// 	setParserV2: value =>({
+		// 				Off : 0,
+		// 				'Fan Mode': value,
+		// 				Reserved : 0,
+		// 	})
+		// });
+
 		this.registerCapability('target_temperature', 'THERMOSTAT_SETPOINT', {
 			get: 'THERMOSTAT_SETPOINT_GET',
 			getOpts: {
 				//getOnStart: true,
+			},
+			set: 'THERMOSTAT_SETPOINT_SET',
+			setParserV2: value =>{
+
+				// Create value buffer
+				const bufferValue = new Buffer(2);
+				bufferValue.writeUInt16BE((Math.round(value * 2) / 2 * 10).toFixed(0));
+
+				return{
+
+					Level: {
+						'Setpoint Type': 'Cooling 1',
```

Comment:
question: you now "hardcoded" a setpoint type to `Cooling 1`...
Wouldn't it be better to use a setpoint type dependent on the thermostat mode?

As in:
`'Setpoint Type': Mode2Setpoint[this.getCapabilityValue('AC_mode') || 'Cooling 1'],

with Mode2Setpoint as defined in:
https://github.com/TedTolboom/no.ThermoFloor/blob/master/drivers/TF_Thermostat/driver.js#L5

In this case it get's the setpoint name corresponding to the current AC_mode and if not available it will use 'Cooling 1'